# CPSC444 Project: EXPLAINING AND PREDICTING SPATIAL VARIATION IN WETLAND N2O EMISSIONS THROUGH MICROBIAL FUNCTIONAL GENE ASSOCIATIONS 

**Data Source:** Bahram, M., Espenberg, M., Pärn, J. et al. Structure and function of the soil microbiome underlying N2O emissions from global wetlands. Nat Commun 13, 1430 (2022). https://doi.org/10.1038/s41467-022-29161-3
https://www.nature.com/articles/s41467-022-29161-3#Sec21

## Available Data

### All Collected & Measured Variables
| Variable                        | Levels / Unit|
|----------|---------------------------------|
| Location  | Coordinates(Latitude/Longitude)<br>Country/Region<br>Regional ISO code<br> |
| Land_use | Raised Bog<br>Fen<br>Forest<br>Hay field<br>Pasture<br>Peat extraction area |
| Climate    | A (rainy tropical)<br>C (temperate)<br>D (boreal) |
| Landscape  | lowland<br>high mountain<br>mountain valley<br>river valley<br>floodplain |
| Vegetation | Mosses<br>Sedges<br>Grasses<br>Herbs<br>Trees<br>Bare Soil |
| Agricultural intensity  | 0 = no agricultural land use (natural mire, swamp, or bog forest)<br>1 = moderate grazing or mowing (once a year or less)<br>2 = intensive grazing or mowing (more than once a year)<br>3 = arable land (directly fertilized or unfertilized)  |
| N2O flux | µg N O-N m h |
| N2 flux  | µg           |
| phyla relative & absolute abundance  | Bacteria; Acidobacteria<br>Bacteria; Proteobacteria; Alphaproteobacteria<br>Bacteria; Actinobacteria<br>Bacteria; Proteobacteria; Deltaproteobacteria<br>Bacteria; Chloroflexi<br>Bacteria; Proteobacteria; Betaproteobacteria<br>Bacteria; Proteobacteria; Gammaproteobacteria<br>Bacteria; Planctomycetes<br>Bacteria; Verrucomicrobia<br>Bacteria; Bacteroidetes<br>Archaea; Euryarchaeota<br>Bacteria; Nitrospirae<br>Bacteria; Firmicutes<br>Bacteria; Gemmatimonadetes<br>Bacteria; Parcubacteria<br>Bacteria; Spirochaetae<br>Bacteria; Ignavibacteriae<br>Eukaryota; Ascomycota<br>Bacteria; Microgenomates<br>Bacteria; Chlamydiae<br>Archaea; Bathyarchaeota<br>Archaea; Thaumarchaeota<br>Bacteria; Latescibacteria<br>Bacteria; Saccharibacteria<br>Eukaryota; Basidiomycota |
| functional (N-cycling) gene abundance | nirK<br>nirS<br>nosZI<br>nosZII<br>bacterial amoA<br>archaeal_amoA<br>nrfA<br>nifH<br>n-dammo<br>coammox  |
| species diversity  | archaeal<br>bacterial<br>fungal  |
| gene diversity & abundance  | archaeal amoA<br>bacterial amoA<br>coammox<br>nirK<br>nirS<br>nrfA<br>nosZI<br>nosZII<br>n-damo<br>nifH  |
| nitrogen metabolism pathways  | ammonia oxidation IV (autotrophic ammonia oxidizers)<br>ammonia oxidation I (aerobic)<br>ammonia assimilation cycle III<br>reduction I (denitrification)<br>nitrite oxidation<br>ammonia oxidation II (anaerobic)<br>L-tryptophan degradation VI (via tryptamine)<br>L-citrulline-nitric oxide cycle<br>N-glucosylnicotinate metabolism<br>hydroxycinnamic acid serotonin amides biosynthesis<br>hydroxycinnamic acid tyramine amides biosynthesis<br>nitrate reduction IV (dissimilatory)<br>spermidine hydroxycinnamic acid conjugates biosynthesis<br>hordatine biosynthesis<br>nitrate reduction VII (denitrification)<br>nitric oxide biosynthesis (plants)<br>ammonia assimilation cycle I<br>nitrifier denitrification	nitrate reduction III (dissimilatory)<br>nitrate reduction VIII (dissimilatory)<br>nitrate reduction IX (dissimilatory)<br>nitrate reduction X (periplasmic, dissimilatory)  |
| Environmental Factors  | MAP<br>MAT |
| Soil Chemistry |  soil temperature (at 10 & 20cm)<br>max temp of warmest month<br>pH<br>Von Post Grade of Decomposition (VPG)<br>organic matter (OrM)<br>water content<br>water table level|
| Soil Nutrients  |P<br>K<br>Ca<br>Mg<br>NO3<br>NH4<br>N<br>C<br>C:N
 
### Supplementary Data
**Supplementary Data 1** Samples used in this study.

**Supplementary Data 2** Correlations between N2O emissions and the relative abundance of all genera uncovered by metagenomes.

**Supplementary Data 3** List of archaeal OTUs showing strongest correlations to N2O fluxes.

**Supplementary Data 4** Correlation between N2O fluxes and Archaeal OGs across global wetland soils.

**Supplementary Data 5** Comparison between the functional gene profiles (MetaCyc and KEGG modules) of N2O taxa (determined using PacBio metabarcoding) that showed strongest positiveand negative correlations with N2O production.

**Supplementary Data 6** List of genomes that contain genes related to ammonia oxidation from a total of 91 archaeal and 116 bacterial genomes.

**Supplementary Data 7** Analysis of MetaCyc nitrogen metabolism pathways in available Archaeal genomes in JGI as off 15-jul-2020.

**Supplementary Data 8** Characteristics of qPCR primer pairs and programs used.
















